tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -1562,6 +1572,104 @@ DisposableEnvironmentVariable ConfigureDisposableEnvironmentVariable(string key,
                     return new DisposableEnvironmentVariable(key, null); // unset env variable
                 }
             }
+
+            IDisposable ConfigureEnvironment()
+            {
+                var requireEnvironmentCheck = RequireEnvironment.Check();
+                switch (kmsProvider)
+                {
+                    case "aws":
+                        {
+                            return new DisposableBundle(
+                                // AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY must be configured for envVariablesSet=true
+                                ConfigureDisposableEnvironmentVariable("AWS_ACCESS_KEY_ID"),
+                                ConfigureDisposableEnvironmentVariable("AWS_SECRET_ACCESS_KEY"),
+                                ConfigureDisposableEnvironmentVariable("AWS_SESSION_TOKEN", onlyUnsetIfNeeded: true));
+                        }
+                    case "gcp":
+                        {
+                            var filesToRemove = new List<DisposableFile>();
+#if DEBUG
+                            // local development:
+
+                            if (expectedEnvironment)
+                            {
+                                // NOTES:
+                                // 1. the full content of credentials exceeds a content's limit presented in windows UI, use a ps script insted:
+                                // [System.Environment]::SetEnvironmentVariable('GOOGLE_APPLICATION_CREDENTIALS_CONTENT', '{ <creds content> }', [EnvironmentVariableTarget]::User)
+                                // or save only { "private_key" : "..", "client_email" : ".." }
+                                // 2. Env GOOGLE_APPLICATION_CREDENTIALS should be set with absolute path, for example: %DRIVERS_TOOLS%\.evergreen\csfle\gcpkms\googleCreds.json
+
+                                requireEnvironmentCheck
+                                    // env variable with creds that we configure on our machines
+                                    .EnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS_CONTENT", isDefined: true)
+                                    // env variable with this file must be created beforehand. The file itself can't be not created yet
+                                    .EnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS", isDefined: true);
+                                var googleCredentialsContent = Environment.GetEnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS_CONTENT");
+                                // the mocked server requires saving credentials into file and place path to this file into GOOGLE_APPLICATION_CREDENTIALS
+                                var googleCredsPath = Environment.GetEnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS");
```

Comment:
`googleCredsPath` => `googleCredentialsPath`

Reply:
this part is removed